### ISSUE TITLE
[Feature Request] make `printTime` also append the date.

### ISSUE THREAD

**Issue by 0xNF** [leisim/logger#140]
the `printTime` function should also print the current date. This would be useful because I am using this package to write files to disk and would like to know what days an event happened on.

**Comment by haarts** [comment]
What printer are you using? 

It is simple to write your own printer. Have a look here: https://github.com/leisim/logger/blob/master/lib/src/printers/prefix_printer.dart

**Comment by 0xNF** [comment]
Yeah I know it's easy to write my own. I just think I shouldn't have to.
